client/wp-admin-scripts/navigation-opt-out/container.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { Button, Modal } from '@wordpress/components';
+
+export default class NavigationOptOutContainer extends Component {
```

Comment:
@samueljseay won't like this 😛

Reply:
No longer exporting as default - happy holidays @samueljseay!